BuildMasterAutomation/Functions/Get-BMVariable.ps1

```diff
@@ -82,9 +84,14 @@ function Get-BMVariable
     Demonstrates how to get all variables from a specific application.
 
     .EXAMPLE
-    Get-BMVariable -Session $session -Variable 'Var' -Application 'www'
+    Get-BMVariable -Session $session -Name 'Var' -Application 'www'
 
     Demonstrates how to get a specific variable from an application.
+
+    .EXAMPLE
+    Get-BMVariable -Session $session -Name 'Var' -Application 'www' -Raw
+
+    Demonstrates how to get a specific variable from an application as a string.

```

Comment:
Improve the description of the example so it's clear that using `-Raw` returns the variable value as an OtterScript string expression rather than a native PowerShell object.